### DISCUSSION TITLE
Homebrew - repeated Touch ID prompts

### DISCUSSION THREAD

**Discussion by caraar12345** [maxgoedjen/secretive#447] [Q&A] [answered]
Hey folks, I've got `git` set up to default to GitHub SSH even when using `https://github.com` using the below `.gitconfig` entry.

```gitconfig
[url "git@github.com:"]
  insteadOf = "https://github.com/"
```

This is great, **but** it means that Homebrew needs me to tap Touch ID about 12 times every time I do `brew update` etc because of all the GitHub repos it pulls from.

Anyone worked out a way by which to require just one Touch ID press, but still keep SSH for all Git comms?

I've tried to press "allow for 5 mins" on the notification, but unfortunately because the next one pops up instantly, it's pretty much impossible to hit in time. As by Touch IDing the next pop up, that 5 minutes is reset 😢 

I'm not averse to simply adding the below if it's not possible though!

```gitconfig
[url "https://github.com/homebrew"]
  insteadOf "git@github.com:homebrew/"
```

**Comment by maxgoedjen** [comment, marked as answer]
@caraar12345 if you click the dropdown on the notification after approving, you can select "unlock for ___" and leave it unlocked for a set period of time.
